ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/mgmt.py

```diff
@@ -54,3 +55,43 @@ def mgmt_gateway(self: SharedUtils) -> str | None:
     @cached_property
     def ipv6_mgmt_gateway(self: SharedUtils) -> str | None:
         return get(self.hostvars, "ipv6_mgmt_gateway")
+
+    @cached_property
+    def default_mgmt_method(self: SharedUtils) -> str | None:
+        """
+        This is only executed if some protocol looks for the default value, so we can raise here to ensure a working config.
```

Comment:
As discussed offline, lets explain here why we only check `ip` for `oob` and only `interface` for `inband` so that it is clear what previous checks we rely on to write this logic